financial-templates-lib/clients/Web3WebsocketClient.js

```diff
@@ -0,0 +1,34 @@
+// This client exports a web3 instances which mimics that created by the default truffle config. It exclusively uses
+// websocket connections to take advantage of reconnect logic if the websocket disconnects or times out. The network
+// syntax mimics that of the main UMA Truffle implementation to make this backwards compatible.
+
+const Web3 = require("web3");
+const { getTruffleConfig, nodeUrl } = require("@umaprotocol/common/TruffleConfig");
+const argv = require("minimist")(process.argv.slice(), { string: ["network"] });
+
+const websocketOptions = {
+  timeout: 10000, // ms
+  clientConfig: {
+    maxReceivedFrameSize: 100000000, // Useful if requests result are large bytes - default: 1MiB
```

Comment:
This might be premature optimization. I added this because the docs spoke about being able to stuff more into each tx. See [here](https://github.com/theturtle32/WebSocket-Node/blob/v1.0.31/docs/WebSocketClient.md#client-config-options) and [here](https://web3js.readthedocs.io/en/v1.2.7/web3.html#configuration) if you want more context.

Reply:
Any downside to increasing max size?

Reply:
Not that I can see? Not sure but I don't think it will cause any issues.